infrateam_first_responder.md

```diff
@@ -1,9 +1,7 @@
 # Guide for Infrastructure Team First Responder
 
-The DLSS infrastructure team is using a rotating role of "first responder".  This doc explains a bit about the concept of
-the first responder role and then outlines specific duties and expectations.
-
-PRs to this doc are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
+The DLSS Infrastructure team is using a rotating role of "first responder." This doc explains the concept of
+the first responder role and outlines specific duties and expectations.
```

Comment:
don't do manual line break here;  we don't in other sections.

Reply:
That was my editor "helping" me. :roll_eyes: Fixed!